app/marketing/mails.py

```diff
@@ -1107,6 +1111,18 @@ def new_bounty_request(model):
             from_name=_("No Reply from Gitcoin.co"),
             categories=['admin', 'new_bounty_request'],
         )
+
+        if model.github_org_email != '':
```

Comment:
![image](https://user-images.githubusercontent.com/1358093/65887348-29e38980-e374-11e9-98bc-43bb0050fcd9.png)
I don't know why if I update my org email (or hide it) the email always keep going to the old one (but correctly save on the table)

Reply:
Also can we put from the server email? To avoid scam gmail message

Reply:
oohh wait I think is just because is from my local is already the server email right?